### PR TITLE
adding taxes page

### PR DESCRIPTION
**Pull request by Almix047**

### CODE REVIEW

File: app/controllers/taxes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class TaxesController < ApplicationController
```
Comment (allomov) [inline app/controllers/taxes_controller.rb]: you can use `rails g scaffold_controller taxes`

File: app/models/tax.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Tax < ApplicationRecord
```
Comment (allomov) [inline app/models/tax.rb]: `belongs_to`?

File: app/views/taxes/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Taxes</h1>
+
+<table border="3" cellspacing="0" cellpadding="10">
```
Comment (allomov) [inline app/views/taxes/index.html.erb]: try to use bootstrap table for that

File: db/migrate/20180806152607_create_taxes.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateTaxes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :taxes do |t|
+      t.string :date
```
Comment (allomov) [inline db/migrate/20180806152607_create_taxes.rb]: as I understand, we need to have a period of dates, when we need to send declaration to a tax inspection. could you please make it a range and name it with `declaration_period` or smth like that.

File: app/views/taxes/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Date:</strong>
+  <%= @tax.date %>
```
Comment (allomov) [inline app/views/taxes/show.html.erb]: period

**Comment by allomov** [comment]
что делать с этим PR?

**Comment by Almix047** [comment]
Closed because it is a duplicate.

**Pull request closed without merging**